meetings/2022-06-16.md

```diff
@@ -43,17 +46,38 @@
 
 #### dokieli
 * SC: [dokieli](https://github.com/linkeddata/dokieli) looks for `solid:storageDescription` (experimenting with this discovery until https://github.com/solid/notifications/issues/58 is resolved) to discover notification channel/subscription information).
-* Sc: Implemented a view for "Storage details" - not particularly interesting/useful to the user in the context of notification stuff, but it was to show that the application is aware of what's available at this point.
+* SC: Implemented a view for "Storage details" - not particularly interesting/useful to the user in the context of notification stuff, but it was to show that the application is aware of what's available at this point.
 * SC: Simple screencast at https://github.com/solid/specification/issues/355#issuecomment-1140508784 along with other findings in the comment.
 * SC: Also implemented subscription request to `WebSocketSubscription2021` and connect to `notify:source`. I didn't yet implement an actual feature to use this - will share that in the future. Need server implementations to move forward in parallel to get anywhere with this. (Also need to have dokieli use Solid OIDC but this is a side issue.)
 * SC: Left comments/issues/PRs based on this.
-
+* eP: If GET or HEAD includes that Link relation on every resource that responds with 2xx response that seems to solve the problem.
+* SC: May not be limited to 2xx.
 
 ### Discovery of Notification Subscription Metadata
 URL: https://github.com/solid/notifications/issues/58
 
 * SC: I'll summarise https://github.com/solid/notifications/issues/58#issuecomment-1151117524 .
-
+* CB: Each resource points to a notification discovery resource or if every resource in a container is managed by the same channel. If I subscribe to a container, I get a notification about the container. Is that intended to be the same or buggy?
```

Comment:
First sentence doesn't parse...

Reply:
@uvdsl , would you like to clarify this?